app/controllers/manage_cmr/order_option_assignments_controller.rb

```diff
@@ -167,4 +303,44 @@ def get_order_options
 
     order_option_select_values.sort
   end
+
+  def get_order_options
+    order_options_response = cmr_client.get_order_options(provider_id: current_user.provider_id, token: token)
+    if order_options_response.error?
+      flash[:error] = I18n.t("controllers.manage_cmr.get_order_option_list.flash.error", request: request.uuid)
+      Rails.logger.error("#{request.uuid} - OrderOptionAssignmentsController#get_order_options - Retrieve Order Options Error, message=#{ex.message}, stacktrace=#{ex.backtrace}")
+      return
+    end
+    order_option_list = Array.wrap(order_options_response.body.fetch('items', []))
+
+    order_option_select_values = []
+
+    order_option_list.select { |option| option.fetch('umm', {}).fetch('Deprecated', false) == false }.each do |order_option|
+      opt = [order_option.fetch('umm', {}).fetch('Name', ''), order_option.fetch('meta', {}).fetch('concept-id', '')]
+      order_option_select_values << opt
+    end
+
+    order_option_select_values.sort
```

Comment:
Is this going to sort by 'Name' or 'concept-id'

Reply:
it sorts by name.